skt/executable.py

```diff
@@ -36,7 +36,8 @@
 import skt.runner
 from skt.kernelbuilder import KernelBuilder, CommandTimeoutError, ParsingError
 from skt.kerneltree import KernelTree, PatchApplicationError
-from skt.misc import SKT_SUCCESS, SKT_FAIL, SKT_ERROR
+from skt.misc import (join_with_slash, get_patch_name, get_patch_mbox,
```

Comment:
If you check the first two commits, there are still two import lines kept.

Reply:
@veruu right. I will fix now